packages/ui/components/Badge/index.tsx

```
@@ -37,18 +45,56 @@
+          {renderContent()}
+        </span>
+      </DropdownMenuTrigger>
+      {options && (
+        <DropdownMenuContent className="mt-1 bg-white shadow-lg">
+          {options.map((option, index) => (
+            <DropdownMenuItem
+              key={index}
+              className={cn("cursor-pointer px-4 py-2 hover:bg-slate-100", textSize)}
+              onClick={(event) => {
+                event.stopPropagation();
+                onChange && onChange(index);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)